must-gather/collection-scripts/gather_crds

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+source pwait
+max_parallelism=10
```

Comment:
capital letters for variable names in shell

Reply:
I think it's not resolved?

`max_parallelism=10` imo should be `MAX_PARALLELISM=10`

Reply:
Ah see it https://github.com/openshift/oadp-operator/issues/1284